ddtrace/contrib/botocore/patch.py

```diff
@@ -70,8 +70,14 @@ def patched_api_call(original_func, instance, args, kwargs):
         result = original_func(*args, **kwargs)
 
         response_meta = result['ResponseMetadata']
-        span.set_tag(http.STATUS_CODE, response_meta['HTTPStatusCode'])
-        span.set_tag('retry_attempts', response_meta['RetryAttempts'])
+
+        http_status_code = response_meta.get('HTTPStatusCode')
+        if http_status_code:
+            span.set_tag(http.STATUS_CODE, http_status_code)
+
+        retry_attempts = response_meta.get('RetryAttempts')
+        if retry_attempts:
```

Comment:
Can we change the checks to be something like

```suggestion
        if 'RetryAttempts' in response_meta:
            span.set_tag('retry_attempts', response_meta['RetryAttempts'])
```

so that we aren't rejecting valid falsy values like `0`, `""`, etc?

Reply:
done in 05347fe 👍